site/includes/sitesearch.hbs

```diff
@@ -1,13 +1,13 @@
-<section id="wb-srch" class="col-lg-8 text-right">
+<section id="wb-srch" class="col-lg-offset-4 col-md-offset-4 col-sm-offset-2 col-xs-12 col-sm-5 col-md-4">
 	<h2>{{{i18n "tmpl-search"}}}</h2>
 	<form action="#" method="post" name="cse-search-box" role="search" class="form-inline">
-		<div class="form-group">
+		<div id="wb-srch-div" class="form-group">
 			<label for="wb-srch-q" class="wb-inv">{{{i18n "tmpl-search-site"}}}</label>
 			<input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="{{{i18n "tmpl-search-site"}}}" />
 			<datalist id="wb-srch-q-ac">
 			</datalist>
 		</div>
-		<div class="form-group submit">
+		<div class="form-group submit sitesearch-submit">
```

Comment:
remove the class here, it is not necessary. Use css selector based on `#wb-srch`